collection.go

```diff
@@ -157,8 +167,6 @@ type CollectionProperties struct {
 	// Schema for collection validation
 	Schema *CollectionSchemaOptions `json:"schema,omitempty"`
 
-	Revision string `json:"revision,omitempty"`
```

Comment:
Both removing ArangoError and Revision fields from struct seems like a breaking change.

e.g. see example:
https://go.dev/play/p/gJ8gF6p4wjY?v=goprev
after moving field into nested object it does not compile:
https://go.dev/play/p/6XOYRBtE4Z5?v=goprev

Reply:
Yes if you will use it to build new object.
Otherwise for read only purpose it has no impact.

Reply:
Yes but we don't know all the cases how our code is used.
It is easy to imagine someone is using something like

```
	props := driver.CollectionProperties{Revision: ""}
	err := resp.ParseBody("", &props)
```

This code obviously is not good and should be refactored but we should not force customers to do it in a such a way.